helm/atlantis/values.yaml

```diff
@@ -0,0 +1,130 @@
+orgWhitelist: github.com/yourorg/*
+
+# If using GitHub, specify like the following:
+# github:
+#   user: foo
+#   token: bar
+#   secret: baz
+# GitHub Enterprise only:
+#   hostname: github.your.org
+# (The chart will perform the base64 encoding for you.)
+
+# If using GitLab, specify like the following:
+# gitlab:
+#   user: foo
+#   token: bar
+#   secret: baz
+# GitLab Enterprise only:
+#   hostname: gitlab.your.org
+# (The chart will perform the base64 encoding for you.)
+
+# If using Bitbucket, specify like the following:
+# bitbucket:
+#   user: jkodroff
+#   token: 7EKP4yWUpLRBJh2pW2t3
+# Bitbucket Server only:
+#   secret: mysecret
+#   base_url: https://bitbucket.2ndwatch.com
+
+
+# When referencing Terraform modules in private repositories, it may be helpful
+# (necessary?) to use redirection in a .gitconfig like so:
+# gitconfig:
+  # [url "https://YOUR_GH_TOKEN@github.com"]
+  #   insteadOf = https://github.com
+  # [url "https://YOUR_GH_TOKEN@github.com"]
+  #   insteadOf = ssh://git@github.com
+  # [url "https://oauth2:YOUR_GITLAB_TOKEN@gitlab.com"]
+  #   insteadOf = https://gitlab.com
+  # [url "https://oauth2:YOUR_GITLAB_TOKEN@gitlab.com"]
+  #   insteadOf = ssh://git@gitlab.com
+# Source: https://stackoverflow.com/questions/42148841/github-clone-with-oauth-access-token
+
+# To specify AWS credentials to be mapped to ~/.aws:
+# aws:
+#   credentials: |
+#     [default]
+#     aws_access_key_id=YOUR_ACCESS_KEY_ID
+#     aws_secret_access_key=YOUR_SECRET_ACCESS_KEY
+#     region=us-east-1
+#   config: |
+#     [profile a_role_to_assume]
+#     role_arn = arn:aws:iam::123456789:role/service-role/roleToAssume
+#     source_profile = default
+
+## To be used for mounting credential files (when using google provider).
+serviceAccountSecrets:
+  # credentials: <json file as base64 encoded string>
+  # credentials-staging: <json file as base64 encoded string>
+
+
+## -------------------------- ##
+# Default values for atlantis (override as needed).
+## -------------------------- ##
+
+image:
+  repository: runatlantis/atlantis
+  tag: v0.4.5
+  pullPolicy: IfNotPresent
+
+## enable using atlantis.yaml file
+allowRepoConfig: false
+
+# We only need to check every 60s since Atlantis is not a high-throughput service.
+livenessProbe:
+  enabled: true
+  periodSeconds: 60
+  initialDelaySeconds: 5
+  timeoutSeconds: 5
+  successThreshold: 1
+  failureThreshold: 5
+  scheme: HTTP
+readinessProbe:
+  enabled: true
+  periodSeconds: 60
+  initialDelaySeconds: 30
+  timeoutSeconds: 5
+  successThreshold: 1
+  failureThreshold: 5
+  scheme: HTTP
+
+service:
+  type: NodePort
+  port: 80
+
+ingress:
+  enabled: true
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  host: chart-example.local
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources:
+  requests:
+    memory: 256Mi
+    cpu: 100m
+  limits:
```

Comment:
We were running with 512Mi at Hootsuite IIRC. Atlantis itself doesn't use many resources, it's the underlying calls to `terraform`. I think we could maybe bump this up to 1GB because someone else was saying they got weird errors when dealing with lots of resources.